pjlib/src/pjlib-test/timer.c

```diff
@@ -334,7 +334,7 @@ static int stress_worker(void *arg)
         idx = pj_rand() % ST_ENTRY_COUNT;
         prev_status = pj_atomic_get(tparam->status[idx]);
         if (task == SCHEDULING) {
-            if (prev_status != 0) continue;
+            //if (prev_status != 0) continue;
```

Comment:
@sauwming This seems to cause a lot of `Warning! Rescheduling outstanding entry (0xxxx)` warnings (about 4000 in my comp) to be printed. Is it okay?